types/react-native/index.d.ts

```diff
@@ -6573,7 +6573,7 @@ export interface ScrollViewPropsIOS {
     pinchGestureEnabled?: boolean;
 
     /**
-     * This controls how often the scroll event will be fired while scrolling (in events per seconds).
+     * This controls how often the scroll event will be fired while scrolling (as a time interval in ms).
      * A higher number yields better accuracy for code that is tracking the scroll position,
      * but can lead to scroll performance problems due to the volume of information being send over the bridge.
```

Comment:
Can you also fix the next sentence? It should be `A lower number yields better accuracy for code that is tracking the scroll position, but can lead to scroll performance problems due to the volume of information being sent over the bridge.` - thanks!

Reply:
Oops! Nice catch, fixed.